cmd/server-main.go

```diff
@@ -600,7 +600,8 @@ func serverMain(ctx *cli.Context) {
 	}
 
 	bootstrapTrace("xhttp.NewServer")
-	httpServer := xhttp.NewServer(getServerListenAddrs()).
+	listenAddrs := getServerListenAddrs()
```

Comment:
If you want current listenAddrs() move this as a function function in `httpServer.ListenAddrs()`  @vadmeste